src/components/application_manager/src/commands/mobile/alert_maneuver_request.cc

```diff
@@ -256,6 +197,54 @@ void AlertManeuverRequest::on_event(const event_engine::Event& event) {
                &(message[strings::msg_params]));
 }
 
+bool AlertManeuverRequest::CalculateResultResultCode(
+    mobile_apis::Result::eType& result_code, std::string& return_info) {
+  LOG4CXX_AUTO_TRACE(logger_);
+  using namespace helpers;
+  bool is_tts_ok;
+  bool is_no_navi_error;
+  hmi_apis::Common_Result::eType tts_result;
+  hmi_apis::Common_Result::eType navi_result;
```

Comment:
@dtrunov 
Do not create uninitialized variables.
https://google.github.io/styleguide/cppguide.html#Local_Variables